### ISSUE TITLE
Remove interstitial pages from end of game play, adjustments to all pages in end path

### ISSUE THREAD

**Issue by alavatelliMCA** [coyote-team/coyote#355]
After entering username and email user is sent to survey page and then submit sends them to the leaderboard. (this removes 2 interstitial pages) This also effects the text that appears on each page, so below I have outlined the text for these pages, each number indicates a new page. Also below I have attached a picture of printouts illustrating the user path. Also, ideally we would like to globally swap "location" out for "site" to gain real estate back with a shorter word.

1. Ending the game
- add a second no option

**Are you sure you want to end the game?
__Yes!__
__No! I want to keep playing!__
__No! I want to keep playing at a new site!__**
note: Sina, all users agreed they would prefer to have the select a new site from the end-the-game page. This would link to the sites (locations) page.

2. "Thanks for Playing" page 
- Header changes to "submit your score"
- Change text below user/email submission form to warn users about ending game

**Submit Your Score
Your email is required to win a prize. We will only contact if you are a winner. Only your username is displayed on the leaderboard.
Username (required)
Email (optional)
When you submit your score you end your game at all site. Instead of ending the game, __change your site__ to keep playing and compete for a grand prize.
__Submit__**

note: change your site is a link to site (locations) page

2. REMOVE THE INTERSTITIAL THANKS FOR PLAYING PAGE
Send to survey instead

3. Survey page
- New header on Survey
- new text below header (pulled from removed page)
- Add leaderboard link below survey
- updated text on the survey page illustrated below
- updated link that surpasses second interstitial page

**Thanks for playing!
Your responses to this survey will help us improve the game.
Was it fun?
Yes
No
Kind of
Was it hard?
Yes
No
Kind of
Would you play it again?
Yes
No
Maybe
What would you change?
__Submit and view the leaderboard__**

4. REMOVE INTERSTITIAL Thank you page post-survey, instead go straight to the leaderboard!

5.  Leaderboard page
- additional text below leaderboard

**Leaderboard**
(leaderboard content)
**Tell your friends about the game. #PlayCoyote
Watch for new clues and game days at coyote.pics/hunt.**

**Comment by stacietaylorcima** [comment]
Loved this picture of the user path - VERY helpful! I'm gonna close this issue cause I just completed all the to-do's, but open any new issues for design fixes you see and I'll get right on them. 

One thing I am thinking: Do you think we should put the "Tell your friends about the game. #PlayCoyote. Watch for new clues and game days at coyote.pics/hunt." above the scores so that in the event that there are a ton of scores, our users still see that message? 

@alavatelliMCA

**Comment by stacietaylorcima** [comment]
Also, what do you think about adding a link to the bottom of the leaderboard that says "Start a new location". The user could click COYOTE to go back through the path, but might be nice to have an explicit link too?

**Comment by alavatelliMCA** [comment]
I think that the question is resolved by the approach to the leaderboard strictly being a global score for when a player is done with all locations. Once a player is done they are done, they can try to play again, but we would only count their first score. It was confusing to have a leaderboard for each location, we actually didn't understand that was happening until yesterday. Originally it was to function like this, but when we removed click to open we confused ourselves #223

**Comment by alavatelliMCA** [comment]
And yes about the message above the scores!

**Comment by stacietaylorcima** [comment]
@alavatellimca That makes total sense with the leaderboard and I’ll move the text today! :-)
